templates/nonOverlappingModules.html

```diff
@@ -0,0 +1,52 @@
+$def with (lst_of_independ_mods, lst_of_ay_sem, ay_sem)
+
+$var title:Non-Overlapping Modules
+
+<div class="container container-fluid">
+    <div class="row">
+        <div class="col-md-12">
+            <div class="row"><!--header-->
+                <h1 class="text-center"><b>Non-Overlapping Modules for $ay_sem</b></h1>
```

Comment:
I think this is ok for now. We'll eventually get a list of what needs to be changed after we start another round of usability heuristic evaluation anyway.